x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/index/signals_mapping.json

```diff
@@ -195,4 +198,4 @@
       }
     }
   }
-}
+}
```

Comment:
nit: newline

Surprised the auto formatter of prettier didn't force one on you